CONTRIBUTING.md

```diff
@@ -0,0 +1,276 @@
+# Contributing to nextcord
+
+First off, thanks for taking the time to contribute. It makes the library substantially better. :+1:
+
+The following is a set of guidelines for contributing to the repository. These are guidelines, not hard rules.
+
+## This is too much to read! I want to ask a question
+
+Generally asking questions are better suited in our resources below.
+
+- [The official support server](https://discord.gg/ZebatWssCB)
+- [The FAQ in the documentation](https://nextcord.readthedocs.io/en/latest/faq.html)
+- [StackOverflow's `nextcord` tag](https://stackoverflow.com/questions/tagged/nextcord)
+- [The discussions page](https://github.com/nextcord/nextcord/discussions)
+
+Please try your best not to ask questions in our issue tracker. Most of them don't belong there unless they provide value to a larger audience.
+
+## Good Bug Reports
+
+Please be aware of the following things when filing bug reports.
+
+1. Don't open duplicate issues. Please search your issue to see if it has been asked already. Duplicate issues will be closed.
+2. When filing a bug about exceptions or tracebacks, please include the *complete* traceback. Without the complete traceback the issue might be **unsolvable** and you will be asked to provide more information.
```

Comment:
```suggestion
2. When filing a bug about exceptions, please include the *complete* traceback. Without the complete traceback the issue might be **unsolvable** and you will be asked to provide more information.
```
Aren't exceptions always presented in tracebacks?

Reply:
I never wrote this, it is from the original contributing.md, but sure.